doc/source/vertical_ref.md

```diff
@@ -1,5 +1,233 @@
+- a {class}`pyproj.CRS<pyproj.crs.CRS>` with only a vertical axis for a CRS based on geoid heights (e.g., the EGM96 geoid).
+
+In practice, a {class}`pyproj.CRS<pyproj.crs.CRS>` with only a vertical axis is either a {class}`~pyproj.crs.BoundCRS` (when created from a grid) or a {class}`~pyproj.crs.VerticalCRS` (when created in any other manner).
+
+## Methods to manipulate vertical CRSs
+
+The parsing, setting and transformation of vertical CRSs revolves around **three methods**, all described in details further below:
+- The **instantiation** of {class}`~xdem.DEM` that implicitly tries to set the vertical CRS from the metadata (or explicitly through the `vcrs` argument),
+- The **setting** method {func}`~xdem.DEM.set_vcrs` to explicitly set the vertical CRS of a {class}`~xdem.DEM`,
+- The **transformation** method {func}`~xdem.DEM.to_vcrs` to explicitly transform the vertical CRS of a {class}`~xdem.DEM`.
+
+```{caution}
+As yet, **[Rasterio](https://rasterio.readthedocs.io/en/stable/) does not support vertical transformations during CRS reprojection** (even if the CRS provided contains a vertical axis).
+We therefore advise to perform horizontal transformation and vertical transformation independently using {func}`DEM.reproject<xdem.DEM.reproject>` and {func}`DEM.to_vcrs<xdem.DEM.to_vcrs>`, respectively.
+```
+
+(vref-setting)=
+## Automated vertical CRS detection
+
+During instantiation of a {class}`~xdem.DEM`, the vertical CRS {attr}`~xdem.DEM.vcrs` is tentatively set with the following priority order:
+
+1. **From the {attr}`~xdem.DEM.crs` of the DEM**, if 3-dimensional,
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+import xdem
+
+# Replace this with a new DEM in xdem-data
+import numpy as np
+import pyproj
+import rasterio as rio
+dem = xdem.DEM.from_array(data=np.ones((2,2)),
+                     transform=rio.transform.from_bounds(0, 0, 1, 1, 2, 2),
+                     crs=pyproj.CRS("EPSG:4326+5773"),
+                     nodata=None)
+dem.save("mydem_with3dcrs.tif")
+```
+
+```{code-cell} ipython3
+import xdem
+
+# Open a DEM with a 3D CRS
+dem = xdem.DEM("mydem_with3dcrs.tif")
+# First, let's look at what was the 3D CRS
+pyproj.CRS(dem.crs)
+```
+
+```{code-cell} ipython3
+# The vertical CRS is extracted automatically
+dem.vcrs
+```
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+import os
+os.remove("mydem_with3dcrs.tif")
+```
+
+2. **From the {attr}`~xdem.DEM.product` name of the DEM**, if parsed from the filename by {class}`geoutils.SatelliteImage`.
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# Replace this with a new DEM in xdem-data
+import rasterio as rio
+dem = xdem.DEM.from_array(data=np.ones((2,2)),
+                     transform=rio.transform.from_bounds(0, 0, 1, 1, 2, 2),
+                     crs=pyproj.CRS("EPSG:4326"),
+                     nodata=None)
+dem.save("SETSM_WV03_20151101_104001001327F500_104001001312DE00_seg2_2m_v3.0_dem.tif")
```

Comment:
Could you add a comment describing the choice of the string here? People might not be aware of that this is the filename pattern of an ArcticDEM. It's written below, but here it's a bit out of the blue.

Reply:
Good idea, added!